static/app/views/insights/common/components/createAlertButton.tsx

```diff
@@ -28,6 +29,7 @@ export function CreateAlertButton({yAxis, search}: Props) {
     ? getAlertsUrl({
         project,
         query: search?.formatString(),
+        dataset: Dataset.EVENTS_ANALYTICS_PLATFORM,
```

Comment:
If this request _always_ goes to EAP, I think you'll need to also update `InsightsTimeSeriesWidget` to only show the alert button if using EAP. Right now it's only checking the `insights-chart-actions` flag

Reply:
That's a fair point, i should check against useEap as well!